phpBB/styles/prosilver/template/navbar_footer.html

```diff
@@ -1,4 +1,4 @@
-<div class="navbar">
+<div class="navbar" role="navigation">
```

Comment:
what about navbar header? if it can only appear once, I'd put it there, since header also contains the forum parents list